etc/openwrt/openthread-br/README.md

```diff
@@ -0,0 +1,44 @@
+OpenThread Border Router on OpenWRT
+===================================
+
+## Get Started
+
+This is for local development.
+
+1. Add OpenThread feed.
+
+Suppose `OPENWRT_TOP_SRCDIR` is root of openwrt sources.
+
+```bash
+echo src-link openthread "$(pwd)/etc/openwrt" >> ${OPENWRT_TOP_SRCDIR}/feeds.conf
+cd "${OPENWRT_TOP_SRCDIR}"
+./scripts/feeds update openthread
+./scripts/feeds install openthread-br
+```
+2. Enable OpenThread.
+
+```bash
+make menuconfig
+```
+
+Check *Network* > *OpenThread Border Router*.
```

Comment:
Is this is some kind of dialog window?  Can you provide more detail as to what happens here and where they check this option.

Reply:
Yes, this is done in a dialog window.  I will add more details.